libmultilabel/data_utils.py

```diff
@@ -158,15 +160,21 @@ def load_or_build_text_dict(
 
     if os.path.exists(embed_file):
         logging.info(f'Load pretrained embedding from file: {embed_file}.')
-        embedding_weights = get_embedding_weights_from_file(vocabs, embed_file, silent, normalize)
-        vocabs.set_vectors(vocabs.stoi, embedding_weights,
-                           dim=embedding_weights.shape[1], unk_init=False)
+        embedding_weights = get_embedding_weights_from_file(vocabs, embed_file, silent)
+        vocabs.set_vectors(vocabs.stoi, embedding_weights, dim=embedding_weights.shape[1])
```

Comment:
how about the unk_init=False? not needed any more

Reply:
According to the document (https://pytorch.org/text/0.9.0/_modules/torchtext/vocab.html#Vocab.set_vectors)
`unk_init` in `set_vectors` should be a callback function here. I'm not sure why we put `False` here.
For caml, removing `unk_init=False` won't the result. That's why I remove it.